docs/csharp/whats-new/csharp-version-history.md

```diff
@@ -94,7 +94,7 @@ With versions 3.0 and 5.0, C# had added some impressive features in an object or
 - [Null propagator](https://davefancher.com/2014/08/14/c-6-0-null-propagation-operator/)
 - [String interpolation](../language-reference/tokens/interpolated.md)
 - [nameof operator](https://stackoverflow.com/questions/31695900/what-is-the-purpose-of-nameof)
-- [Dictionary initializer](../programming-guide/classes-and-structs/how-to-initialize-a-dictionary-with-a-collection-initializer.md)
+- [Dictionary initializer](csharp-6.md#index-initializers)
```

Comment:
@SheepRock Thank you for fixing that! To make the whole link correct, I suggest to change the link text as well to "Index initializers* as initializing dictionary is a specific use case of applying index initializers.